src/styles/page-template/custom/code.njk

```diff
@@ -50,6 +51,33 @@ ignore_in_sitemap: true
 {% endblock %}
 
 {% block bodyStart %}
+  {{ govukCookieBanner({
+    ariaLabel: "Cookies on [name of service]",
+    messages: [
+      {
+        headingText: "Cookies on [name of service]",
+        html: html,
```

Comment:
`html` isn't defined anywhere, so the cookie banner only has a heading (and buttons).

Reply:
👍  I've added it as multi-line content instead of setting a variable to keep the files tidier and to make it clearer what page template variables/blocks are being set.